src/internal/account.ts

```diff
@@ -169,6 +169,47 @@ export async function getResource(args: {
   return data;
 }
 
+export async function lookupOriginalAccountAddress(args: {
+  aptosConfig: AptosConfig;
+  addressOrAuthKey: HexInput;
```

Comment:
We should just rename this to `authenticationKey`, you technically always lookup an address by the auth key